src/sentry/static/sentry/app/utils/events.tsx

```diff
@@ -54,7 +54,10 @@ export function getTitle(event: Event | Group): EventTitle {
     result.title = metadata.directive || '';
     result.subtitle = metadata.uri || '';
   } else if (type === 'expectct' || type === 'expectstaple' || type === 'hpkp') {
-    result.title = metadata.message || '';
+    // Due to a regression some reports did not have message persisted
+    // (https://github.com/getsentry/sentry/pull/19794) so we need to fall
+    // back to the computed title for these.
```

Comment:
Should we have a TODO to remove this in 100 days?

Reply:
I let a note for myself. @matejminar and me have been talking about some refactoring on this entire title stuff for #21297.

Reply:
Are we actually able to remove this, or will it potentially be broken in self hosted for a long time?